projects/1.18/assets/iron-furnaces/ironfurnaces/lang/zh_cn.json

```
@@ -59,6 +68,11 @@
   "container.ironfurnaces.silver_furnace": "银熔炉",
   "container.ironfurnaces.million_furnace": "彩虹熔炉",
   "container.ironfurnaces.wireless_energy_heater": "无线热能供应器",
+  "ironfurnaces.jei_category_regular": "熔炉组件：发电机升级",
+  "ironfurnaces.jei_category_blasting": "发电机|高炉",
+  "ironfurnaces.jei_category_smoking": "发电机|烟熏炉",

```

```suggestion
  "ironfurnaces.jei_category_blasting": "发电机丨高炉",
  "ironfurnaces.jei_category_smoking": "发电机丨烟熏炉",
```
丨（gun3）
就是不知道这里这样改行不行（）

---

完全可以，原来还能这么做!!!∑(ﾟДﾟノ)ノ